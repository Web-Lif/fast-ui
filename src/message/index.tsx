import message from 'antd/es/message'

export default message
