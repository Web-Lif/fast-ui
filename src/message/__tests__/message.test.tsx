import { render } from '@testing-library/react'
import React from 'react'
import message from '..'

const TestMessage = () => {
    const { _InternalPanelDoNotUseOrYouWillBeFired: InternalPanel } = message
    return <InternalPanel content="testmessage" type="error" />
}

test('test message', async () => {
    const { baseElement, findAllByText } = render(<TestMessage />)
    await findAllByText('testmessage')
    expect(baseElement).toMatchSnapshot()
})
