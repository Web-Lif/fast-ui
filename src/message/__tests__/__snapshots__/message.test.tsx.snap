// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test message 1`] = `
<body>
  <div>
    <div />
  </div>
  <div
    class="ant-message ant-message-top css-dev-only-do-not-override-1dsa5t3"
    style="left: 50%; transform: translateX(-50%); top: 8px;"
  >
    <div
      class="ant-message-notice ant-message-notice-info css-dev-only-do-not-override-1dsa5t3"
    >
      <div
        class="ant-message-notice-content"
      >
        <div
          class="ant-message-custom-content ant-message-info"
        >
          <span
            aria-label="info-circle"
            class="anticon anticon-info-circle"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="info-circle"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm32 664c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V456c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v272zm-32-344a48.01 48.01 0 010-96 48.01 48.01 0 010 96z"
              />
            </svg>
          </span>
          <span>
            testmessage
          </span>
        </div>
      </div>
    </div>
  </div>
</body>
`;
